### DISCUSSION TITLE
Reply function while forwarding

### DISCUSSION THREAD

**Discussion by alanliew88** [MrMissx/Telegram_Forwarder#35] [FAQ]
Is this scripts enable to forward (copy) the reply function in the source group too ?

**Comment by MrMissx** [comment]
I don't get what do you mean. Can you explain it clearly?

**Reply by alanliew88** [reply]
Hi sorry, I mean is the reply message function being able to forward to our desired destinated channel / group too ?

**Reply by 19521790** [reply]
i create new sqlite3 database for myself, then i can store desination_reply_message_id of source_reply_message_id. just simple edit file auto_forward.py. Below is my code

[auto_forward.txt](https://github.com/MrMissx/Telegram_Forwarder/files/9585385/auto_forward.txt)
